### PR TITLE
Added in files for jitpack

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-3841

### Change description ###
Added in files for jitpack

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,8 +65,15 @@ compileTestJava {
...
 
 repositories {
+    mavenCentral()
     mavenLocal()
     jcenter()
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+
+    maven {
+        url 'https://repo.spring.io/libs-milestone'
```
Comment (benouaer) [inline build.gradle:75]: why is this now needed?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
```
Comment (benouaer) [inline .github/dependabot.yml:6]: does dependabot run daily now?

Comment (yogesh-hullatti) [reply .github/dependabot.yml]: Updated to weekly now

File: build.gradle (hunk 1 above)
Comment (yogesh-hullatti) [reply build.gradle:75]: Referring to the notes on the slack channel and the example. Seems we need these now. 
https://github.com/hmcts/sscs-common

**Pull request merged**